includes/class-alma-wc-payment-gateway.php

```diff
@@ -30,167 +55,368 @@ public function __construct() {
 		$this->title       = $this->get_option( 'title' );
 		$this->description = $this->get_option( 'description' );
 
-		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, array(
-			$this,
-			'process_admin_options'
-		) );
+		add_action(
+			'woocommerce_update_options_payment_gateways_' . $this->id,
+			array( $this, 'process_admin_options' )
+		);
+
+		add_filter(
+			'woocommerce_settings_api_sanitized_fields_' . $this->id,
+			array( $this, 'on_settings_save' )
+		);
+	}
+
+	/**
+	 * Get option from DB.
+	 *
+	 * Gets an option from the settings API, using defaults if necessary to prevent undefined notices.
```

Comment:
```suggestion
	 * Gets an option from the settings API, using defaults if necessary to prevent undefined notices.
	 * This is overriden so that values saved in cents in the DB can be shown in euros to the user	 
```